db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/MONGETCONNECTIONRowFactory.java

```diff
@@ -0,0 +1,2429 @@
+    protected static final int      TOTAL_BACKUP_TIME = 340;
+    protected static final int      TOTAL_BACKUP_PROC_TIME = 341;
+    protected static final int      TOTAL_BACKUPS = 342;
+    protected static final int      TOTAL_INDEX_BUILD_TIME = 343;
+    protected static final int      TOTAL_INDEX_BUILD_PROC_TIME = 344;
+    protected static final int      TOTAL_INDEXES_BUILT = 345;
+    protected static final int      EXT_TABLE_RECV_WAIT_TIME = 346;
+    protected static final int      EXT_TABLE_RECVS_TOTAL = 347;
+    protected static final int      EXT_TABLE_RECV_VOLUME = 348;
+    protected static final int      EXT_TABLE_READ_VOLUME = 349;
+    protected static final int      EXT_TABLE_SEND_WAIT_TIME = 350;
+    protected static final int      EXT_TABLE_SENDS_TOTAL = 351;
+    protected static final int      EXT_TABLE_SEND_VOLUME = 352;
+    protected static final int      EXT_TABLE_WRITE_VOLUME = 353;
+    protected static final int      FCM_TQ_RECV_WAITS_TOTAL = 354;
+    protected static final int      FCM_MESSAGE_RECV_WAITS_TOTAL = 355;
+    protected static final int      FCM_TQ_SEND_WAITS_TOTAL = 356;
+    protected static final int      FCM_MESSAGE_SEND_WAITS_TOTAL = 357;
+    protected static final int      FCM_SEND_WAITS_TOTAL = 358;
+    protected static final int      FCM_RECV_WAITS_TOTAL = 359;
+    protected static final int      COL_VECTOR_CONSUMER_OVERFLOWS = 360;
+    protected static final int      TOTAL_COL_SYNOPSIS_TIME = 361;
+    protected static final int      TOTAL_COL_SYNOPSIS_PROC_TIME = 362;
+    protected static final int      TOTAL_COL_SYNOPSIS_EXECUTIONS = 363;
+    protected static final int      COL_SYNOPSIS_ROWS_INSERTED = 364;
+    protected static final int      LOB_PREFETCH_WAIT_TIME = 365;
+    protected static final int      LOB_PREFETCH_REQS = 366;
+    protected static final int      FED_ROWS_DELETED = 367;
+    protected static final int      FED_ROWS_INSERTED = 368;
+    protected static final int      FED_ROWS_UPDATED = 369;
+    protected static final int      FED_ROWS_READ = 370;
+    protected static final int      FED_WAIT_TIME = 371;
+    protected static final int      FED_WAITS_TOTAL = 372;
+    protected static final int      APPL_SECTION_INSERTS = 373;
+    protected static final int      APPL_SECTION_LOOKUPS = 374;
+    protected static final int      CONNECTION_REUSABILITY_STATUS = 375;
+    protected static final int      REUSABILITY_STATUS_REASON = 376;
+    protected static final int      ADM_OVERFLOWS = 377;
+    protected static final int      ADM_BYPASS_ACT_TOTAL = 378;
+
+    private	static	final	String[]	uuids = {
+            MONGETCONNECTION_UUID,                    // catalog UUID
+            "64241859-c802-4abb-8ce5-7926d7ac663a",   // heap UUID
+            "3b6c9d6f-5af1-44b3-bb6f-44dfd1ffdacf"    // MONGETCONNECTION_INDEX1_ID
+    };
+
+    protected static final int MONGETCONNECTION_INDEX1_ID = 0;
+
+    private	static	final	boolean[]	uniqueness = {
+            true
```

Comment:
Is APPLICATION_HANDLE unique?

Reply:
According to DB2 documentation, it should be a system-wide unique ID. Reference: https://www.ibm.com/support/knowledgecenter/SSEPGG_10.5.0/com.ibm.db2.luw.admin.mon.doc/doc/monitor-elements-a.html#r0054410